cmd/main.go

```diff
@@ -24,6 +24,7 @@ var config struct {
 	skipAnnotation   bool
 	development      bool
 	zapOpts          zap.Options
+	webhookEnabled   bool
```

Comment:
To make it consistent with the Helm chart, it should be `pvcMutatingWebhookEnabled`.

Reply:
This is consistent with other fields in the config struct itself however:
ie.
`webhookAddr`

hence the decision to use webhookEnabled instead of a field which was consistent with the helm value. 🤔 
With that context, would you still prefer this to be consistent with the helm value? @daichimukai 

Thanks

Reply:
@daichimukai just bumping this, are you happy with this remaining as `webhookEnabled` to be consistent with the config `webhookAddr` - or would you rather it be `pvcMutatingWebhookEnabled` per your original comment? Thanks

Reply:
In the future, we might implement other webhooks besides PVC mutation. It would be nice if we could turn each webhook on or off individually, so I think pvcMutatingWebhookEnabled is preferred. Even then, all webhooks share the same address, so the naming of webhookAddr is okay.

Reply:
Sounds good @daichimukai, I've pushed https://github.com/topolvm/pvc-autoresizer/pull/272/commits/c63546abf9f57962f26c5d98de628aab4e545698 changing the naming to your suggestion.
Thanks!